apps/air-discount-scheme/web/screens/Subsidy/components/Usage/Usage.tsx

```diff
@@ -43,10 +43,9 @@ function Usage({ misc }: PropTypes) {
     ssr: false,
     pollInterval: THIRTY_SECONDS,
   })
-  const {
-    user: { flights },
-  } = data ?? { user: { flights: [] } }
-  const { currentUsage, user, path, date } = JSON.parse(misc)
+  const { user } = data || {}
```

Comment:
wouldn't it be better to use nullish coalescing operator for defaults?

Reply:
I can only see the value there for numbers and strings.
Don't think we gain anything by using the nullish coalescing operator for objects.